aqueduct/kubernetes/api-deployment-and-service.yaml

```diff
@@ -6,10 +6,11 @@ metadata:
 spec:
   selector:
     app: <APP_NAME>
-    tier: frontend
+    tier: backend
```

Comment:
change to role: backend to match angular deployment?

Reply:
👍